### ISSUE TITLE
Many plugins fail to sync with backups

### ISSUE THREAD

**Issue by timlinux** [qgis/QGIS-Django#112]
@sumandari please investigate why and fix the root cause as well as tidying the permissions of these files

```
Total: 57 directories, 140 files, 0 symlinks
New: 6 files, 0 symlinks
144775715 bytes transferred in 476 seconds (296.7 KiB/s)
To be removed: 5 directories, 53 files, 0 symlinks
2 errors detected
Source : /mnt/HC_Volume_4113256/packages
Destination : static
Objective : DOWNLOAD
mirror: Access failed: Permission denied (DSGTools_Op_Deffv37.zip)
mirror: Access failed: Permission denied (DsgTools.zip)
mirror: Access failed: Permission denied (Floodrisk-0.5.zip)
mirror: Access failed: Permission denied (Floodrisk.zip)
mirror: Access failed: Permission denied (FragScape_yi20G97.zip)
mirror: Access failed: Permission denied (LDMP_QGIS3_gGaOGwc.zip)
mirror: Access failed: Permission denied (Mascaret_3.0.2.zip)
mirror: Access failed: Permission denied (Mascaret_3.0.4.zip)
mirror: Access failed: Permission denied (Mascaret_3.0.5.zip)
mirror: Access failed: Permission denied (PostTelemac_1_1_5.zip)
mirror: Access failed: Permission denied (QGISRed_v0.11.zip)
mirror: Access failed: Permission denied (QGeoloGIS-1.4.0.zip)
mirror: Access failed: Permission denied (QGeoloGIS-1.5.0.zip)
mirror: Access failed: Permission denied (QTRANUS_2020_08_03.zip)
mirror: Access failed: Permission denied (QTRANUS_4EkxvSm.zip)
mirror: Access failed: Permission denied (QTRANUS_AIy1ptN.zip)
mirror: Access failed: Permission denied (QTRANUS_C5qjcrU.zip)
mirror: Access failed: Permission denied (QTRANUS_DKTzkJs.zip)
mirror: Access failed: Permission denied (QTRANUS_GLEK5q3.zip)
mirror: Access failed: Permission denied (QTRANUS_JaXNpZ2.zip)
mirror: Access failed: Permission denied (QTRANUS_SRbuWzw.zip)
mirror: Access failed: Permission denied (QTRANUS_XwrJyNc.zip)
mirror: Access failed: Permission denied (QTRANUS_grXrIHb.zip)
mirror: Access failed: Permission denied (QTRANUS_hADbxLX.zip)
mirror: Access failed: Permission denied (QTRANUS_oKY1uOk.zip)
mirror: Access failed: Permission denied (QTRANUS_rDer3wO.zip)
mirror: Access failed: Permission denied (Redistricting.zip)
mirror: Access failed: Permission denied (Redistricting_DYDUJKf.zip)
mirror: Access failed: Permission denied (SSAM-1.0.2.zip)
mirror: Access failed: Permission denied (StreetAnalysis.zip)                        
mirror: Access failed: Permission denied (StreetAnalysis_JNYJtOu.zip)
mirror: Access failed: Permission denied (StreetAnalysis_LvLcxPj.zip)
mirror: Access failed: Permission denied (StreetAnalysis_NwRpQP8.zip)
mirror: Access failed: Permission denied (StreetAnalysis_Y2bNF7O.zip)
mirror: Access failed: Permission denied (StreetAnalysis_oM6YZ9V.zip)
mirror: Access failed: Permission denied (UMEP_ano2vf2.zip)
mirror: Access failed: Permission denied (UMEP_uMxipu8.zip)
mirror: Access failed: Permission denied (UrbanSARCA.zip)
mirror: Access failed: Permission denied (UrbanSARCA_X2UMmdr.zip)
mirror: Access failed: Permission denied (UrbanSARCA_g89ddqj.zip)
mirror: Access failed: Permission denied (accuracy_assessment.zip)
mirror: Access failed: Permission denied (arlula_HmGiVN6.zip)
mirror: Access failed: Permission denied (arlula_Qrkx6o9.zip)
mirror: Access failed: Permission denied (arlula_RmNei3p.zip)
mirror: Access failed: Permission denied (arlula_jznoOHI.zip)
mirror: Access failed: Permission denied (clusterpoints_T9DjReG.zip)
mirror: Access failed: Permission denied (clusterpoints_aj4eS7V.zip)
mirror: Access failed: Permission denied (clusterpoints_u6bm04p.zip)
mirror: Access failed: Permission denied (ee_plugin-0.0.3.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20200908T211050.develop.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20200925T105253.TEST.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201002T173515.develop.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201009T224833.develop.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201023T004415.develop.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201023T182737.develop.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201026T172128.develop.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201028T010538.master.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201028T072145.master.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201029T143643.master.zip)
mirror: Access failed: Permission denied (ezprinter_CHzesXj.zip)
mirror: Access failed: Permission denied (ezprinter_GbNrUmT.zip)
mirror: Access failed: Permission denied (grid.zip)                     
mirror: Access failed: Permission denied (grid_keaTMJm.zip)
mirror: Access failed: Permission denied (huntregister_3ZiREsz.zip)
mirror: Access failed: Permission denied (inde.zip)        
mirror: Access failed: Permission denied (mergin_6S66QUX.zip)
mirror: Access failed: Permission denied (mergin_BTC5Zzu.zip)
mirror: Access failed: Permission denied (mergin_LMucYAB.zip)
mirror: Access failed: Permission denied (mergin_NNjdcbL.zip)
mirror: Access failed: Permission denied (mergin_bvAUok5.zip)
mirror: Access failed: Permission denied (mergin_vX089lu.zip)
mirror: Access failed: Permission denied (minimum_spanning_tree.zip)
mirror: Access failed: Permission denied (mon_parcellaire_125.zip)
mirror: Access failed: Permission denied (network_gt_Duq2m7C.zip)
mirror: Access failed: Permission denied (network_gt_F1odaH1.zip)
mirror: Access failed: Permission denied (network_gt_NHUyYlw.zip)
mirror: Access failed: Permission denied (network_gt_OwS2O1j.zip)
mirror: Access failed: Permission denied (network_gt_fBtIMLi.zip)
mirror: Access failed: Permission denied (network_gt_koFhTql.zip)
mirror: Access failed: Permission denied (network_gt_rhoWOql.zip)
mirror: Access failed: Permission denied (network_gt_scjUXzB.zip)
mirror: Access failed: Permission denied (network_gt_xWYOOu6.zip)
mirror: Access failed: Permission denied (networks_JnssBAV.zip)
mirror: Access failed: Permission denied (networks_MaZFXP9.zip)
mirror: Access failed: Permission denied (networks_h9D1tfV.zip)
mirror: Access failed: Permission denied (pat_v0.3.4_0SYX50g.zip)
mirror: Access failed: Permission denied (pat_v0.3.5.zip)
mirror: Access failed: Permission denied (pat_v1.0.0.zip)
mirror: Access failed: Permission denied (pat_v1.0.0_n1xxAlV.zip)
mirror: Access failed: Permission denied (pat_v1.0.1.zip)
mirror: Access failed: Permission denied (poi_visibility_network.zip)
mirror: Access failed: Permission denied (projektcheck.zip)
mirror: Access failed: Permission denied (projektcheck_JlipnjN.zip)
mirror: Access failed: Permission denied (projektcheck_SN7QIQ2.zip)
mirror: Access failed: Permission denied (qgis-maptiler-plugin_PGFWOgy.zip)
mirror: Access failed: Permission denied (svir_VPVe1ui.zip)   
mirror: Access failed: Permission denied (timeseriesviewerplugin.1.13.20200723T144208.master.zip)
mirror: Access failed: Permission denied (timeseriesviewerplugin.1.13.20200813T152403.master.zip)
mirror: Access failed: Permission denied (timeseriesviewerplugin.1.14.20201107T012304.master.zip)
mirror: Access failed: Permission denied (volume_calculation_tool.zip)
mirror: Access failed: Permission denied (vx_integration.zip)
Total: 7 directories, 20689 files, 0 symlinks          
New: 148 files, 0 symlinks
14131964 bytes transferred in 30 seconds (460.4 KiB/s)
To be removed: 13 directories, 50 files, 0 symlinks
102 errors detected
```

**Comment by timlinux** [comment]
Also it is strange, we seem to have two copies of plugins? Can you check what is going on here?
´´´
root@5e44428e3a92:/usr# cd ..
root@5e44428e3a92:/# find . -name DSGTools_Op_Deffv37.zip
./home/web/media/packages/2020/DSGTools_Op_Deffv37.zip
./home/web/static/packages/2020/DSGTools_Op_Deffv37.zip
root@5e44428e3a92:/#
´´´

**Comment by timlinux** [comment]
You can see here that for some reason the affected plugins are not readable by others

```
root@5e44428e3a92:/home/web/media/packages/2020# ls -lah DSG*
-rw-r--r-- 1 1000 1000  11M Feb 19  2020 DSGToolsOp-master.zip
-rw-r--r-- 1 1000 1000  12M Jan 20  2020 DSGTools_Op.zip
-rw-r--r-- 1 1000 1000 5.7M Jan 20  2020 DSGTools_Op_2.0.zip
-rw-r--r-- 1 1000 1000 5.7M Jan 20  2020 DSGTools_Op_2.0_JHk9lpn.zip
-rw-r--r-- 1 1000 1000  13M Jan 20  2020 DSGTools_Op_2.0_ma1pY1F.zip
-rw------- 1 root root  11M Sep 17 19:33 DSGTools_Op_Deffv37.zip
```

**Comment by timlinux** [comment]
If I run this in a rancher console I can see that it is the same for all affected media:


```

root@5e44428e3a92:/home/web/media/packages/2020# ls -lah | grep "\-\-\-\-\-\-"
-rw------- 1 root root   11M Sep 17 19:33 DSGTools_Op_Deffv37.zip
-rw------- 1 root root   13M Oct 27 21:51 DsgTools.zip
-rw------- 1 root root  5.2M Oct 30 10:07 Floodrisk-0.5.zip
-rw------- 1 root root   14M Aug 20 16:07 Floodrisk.zip
-rw------- 1 root root  9.8M Nov 17 19:07 FragScape_yi20G97.zip
-rw------- 1 root root  3.7M Aug 14 18:00 LDMP_QGIS3_gGaOGwc.zip
-rw------- 1 root root  6.6M Jul 31 14:50 Mascaret_3.0.2.zip
-rw------- 1 root root  6.6M Sep  1 16:07 Mascaret_3.0.4.zip
-rw------- 1 root root  6.6M Sep  4 15:47 Mascaret_3.0.5.zip
-rw------- 1 root root  7.5M Sep 17 15:23 PostTelemac_1_1_5.zip
-rw------- 1 root root  3.8M Sep 15 16:01 QGISRed_v0.11.zip
-rw------- 1 root root   14M Sep  3 18:43 QGeoloGIS-1.4.0.zip
-rw------- 1 root root   14M Sep  3 18:45 QGeoloGIS-1.5.0.zip
-rw------- 1 root root  9.8M Aug  3 15:12 QTRANUS_2020_08_03.zip
-rw------- 1 root root   10M Oct  5 02:32 QTRANUS_4EkxvSm.zip
-rw------- 1 root root   10M Sep  1 14:20 QTRANUS_AIy1ptN.zip
-rw------- 1 root root   10M Oct 21 20:08 QTRANUS_C5qjcrU.zip
-rw------- 1 root root   10M Oct  3 15:32 QTRANUS_DKTzkJs.zip
-rw------- 1 root root   10M Oct  3 17:29 QTRANUS_GLEK5q3.zip
-rw------- 1 root root   10M Aug 30 21:17 QTRANUS_JaXNpZ2.zip
-rw------- 1 root root   10M Oct 26 22:41 QTRANUS_SRbuWzw.zip
-rw------- 1 root root   10M Sep 26 17:23 QTRANUS_XwrJyNc.zip
-rw------- 1 root root   10M Oct  7 23:44 QTRANUS_grXrIHb.zip
-rw------- 1 root root   10M Sep 24 17:01 QTRANUS_hADbxLX.zip
-rw------- 1 root root   10M Oct 25 14:24 QTRANUS_oKY1uOk.zip
-rw------- 1 root root   10M Nov  9 04:41 QTRANUS_rDer3wO.zip
-rw------- 1 root root  5.6M Sep  5 00:34 Redistricting.zip
-rw------- 1 root root  5.6M Sep  9 10:12 Redistricting_DYDUJKf.zip
-rw------- 1 root root   15M Jul 18 08:22 SSAM-1.0.2.zip
-rw------- 1 root root  2.9M Nov  9 12:59 StreetAnalysis.zip
-rw------- 1 root root  2.9M Nov 13 16:14 StreetAnalysis_JNYJtOu.zip
-rw------- 1 root root  2.9M Nov 13 16:12 StreetAnalysis_LvLcxPj.zip
-rw------- 1 root root  2.9M Nov  9 13:00 StreetAnalysis_NwRpQP8.zip
-rw------- 1 root root  2.9M Nov 13 15:59 StreetAnalysis_Y2bNF7O.zip
-rw------- 1 root root  2.9M Nov 13 16:16 StreetAnalysis_oM6YZ9V.zip
-rw------- 1 root root  2.7M Aug 19 09:27 UMEP_ano2vf2.zip
-rw------- 1 root root  2.9M Oct  7 09:54 UMEP_uMxipu8.zip
-rw------- 1 root root  2.7M Oct  8 07:14 UrbanSARCA.zip
-rw------- 1 root root  2.7M Oct  9 06:58 UrbanSARCA_X2UMmdr.zip
-rw------- 1 root root  2.7M Oct 21 20:55 UrbanSARCA_g89ddqj.zip
-rw------- 1 root root   11M Sep 11 20:49 accuracy_assessment.zip
-rw------- 1 root root  8.8M Nov 12 05:03 arlula_HmGiVN6.zip
-rw------- 1 root root  8.7M Sep 30 22:44 arlula_Qrkx6o9.zip
-rw------- 1 root root  8.8M Oct 29 00:21 arlula_RmNei3p.zip
-rw------- 1 root root   11M Oct 22 00:52 arlula_jznoOHI.zip
-rw------- 1 root root  7.3M Aug 29 16:39 clusterpoints_T9DjReG.zip
-rw------- 1 root root  7.3M Aug 29 16:32 clusterpoints_aj4eS7V.zip
-rw------- 1 root root  7.3M Aug 29 16:36 clusterpoints_u6bm04p.zip
-rw------- 1 root root   14M Nov  4 20:21 ee_plugin-0.0.3.zip
-rw------- 1 root root  6.6M Sep 24 19:37 enmapboxplugin.3.7.20200908T211050.develop.zip
-rw------- 1 root root  6.6M Sep 25 08:55 enmapboxplugin.3.7.20200925T105253.TEST.zip
-rw------- 1 root root  6.7M Oct  2 15:53 enmapboxplugin.3.7.20201002T173515.develop.zip
-rw------- 1 root root  7.3M Oct 11 19:24 enmapboxplugin.3.7.20201009T224833.develop.zip
-rw------- 1 root root  7.1M Oct 23 09:44 enmapboxplugin.3.7.20201023T004415.develop.zip
-rw------- 1 root root  7.0M Oct 23 16:32 enmapboxplugin.3.7.20201023T182737.develop.zip
-rw------- 1 root root  7.1M Oct 26 16:41 enmapboxplugin.3.7.20201026T172128.develop.zip
-rw------- 1 root root  7.1M Oct 28 00:06 enmapboxplugin.3.7.20201028T010538.master.zip
-rw------- 1 root root  7.1M Oct 28 07:34 enmapboxplugin.3.7.20201028T072145.master.zip
-rw------- 1 root root  7.2M Oct 29 13:38 enmapboxplugin.3.7.20201029T143643.master.zip
-rw------- 1 root root  3.3M Nov  2 12:47 ezprinter_CHzesXj.zip
-rw------- 1 root root  3.3M Nov  2 13:01 ezprinter_GbNrUmT.zip
-rw------- 1 root root  4.9M Aug 18 22:38 grid.zip
-rw------- 1 root root  4.9M Aug 18 22:53 grid_keaTMJm.zip
-rw------- 1 root root  6.7M Aug 23 11:56 huntregister_3ZiREsz.zip
-rw------- 1 root root   11M Nov 19 19:05 inde.zip
-rw------- 1 root root  4.7M Oct  1 09:40 mergin_6S66QUX.zip
-rw------- 1 root root  4.7M Sep  8 15:37 mergin_BTC5Zzu.zip
-rw------- 1 root root  4.7M Sep  9 20:24 mergin_LMucYAB.zip
-rw------- 1 root root  4.7M Nov 13 08:23 mergin_NNjdcbL.zip
-rw------- 1 root root  4.7M Jul 15 12:00 mergin_bvAUok5.zip
-rw------- 1 root root  4.7M Nov  3 11:37 mergin_vX089lu.zip
-rw------- 1 root root   11M Oct 26 16:16 minimum_spanning_tree.zip
-rw------- 1 root root  8.8M Jul 23 21:47 mon_parcellaire_125.zip
-rw------- 1 root root  6.3M Oct 15 13:37 network_gt_Duq2m7C.zip
-rw------- 1 root root  6.3M Sep 15 10:02 network_gt_F1odaH1.zip
-rw------- 1 root root  6.3M Oct 21 11:28 network_gt_NHUyYlw.zip
-rw------- 1 root root  6.3M Nov 17 17:05 network_gt_OwS2O1j.zip
-rw------- 1 root root  6.3M Oct 15 13:11 network_gt_fBtIMLi.zip
-rw------- 1 root root  6.3M Nov 17 16:59 network_gt_koFhTql.zip
-rw------- 1 root root  6.4M Nov 17 16:11 network_gt_rhoWOql.zip
-rw------- 1 root root  6.3M Nov 17 17:03 network_gt_scjUXzB.zip
-rw------- 1 root root  6.3M Oct 15 13:25 network_gt_xWYOOu6.zip
-rw------- 1 root root   19M Sep 22 15:10 networks_JnssBAV.zip
-rw------- 1 root root   19M Nov  3 07:50 networks_MaZFXP9.zip
-rw------- 1 root root   19M Aug 20 08:29 networks_h9D1tfV.zip
-rw------- 1 root root  7.5M Aug 14 08:04 pat_v0.3.4_0SYX50g.zip
-rw------- 1 root root  7.3M Aug 31 00:34 pat_v0.3.5.zip
-rw------- 1 root root  4.6M Aug 31 04:37 pat_v1.0.0.zip
-rw------- 1 root root  4.6M Aug 31 08:38 pat_v1.0.0_n1xxAlV.zip
-rw------- 1 root root  4.6M Sep  1 05:46 pat_v1.0.1.zip
-rw------- 1 root root   15M Sep 14 05:59 poi_visibility_network.zip
-rw------- 1 root root   11M Jul 31 08:03 projektcheck.zip
-rw------- 1 root root   12M Nov 20 15:02 projektcheck_JlipnjN.zip
-rw------- 1 root root   12M Sep 21 11:45 projektcheck_SN7QIQ2.zip
-rw------- 1 root root  2.8M Jul 20 09:06 qgis-maptiler-plugin_PGFWOgy.zip
-rw------- 1 root root   15M Oct  1 14:48 svir_VPVe1ui.zip
-rw------- 1 root root  4.8M Jul 23 22:04 timeseriesviewerplugin.1.13.20200723T144208.master.zip
-rw------- 1 root root  4.8M Aug 13 13:26 timeseriesviewerplugin.1.13.20200813T152403.master.zip
-rw------- 1 root root  4.8M Nov  7 00:23 timeseriesviewerplugin.1.14.20201107T012304.master.zip
-rw------- 1 root root  6.9M Oct 13 15:39 volume_calculation_tool.zip
-rw------- 1 root root  2.6M Sep  7 10:27 vx_integration.zip
root@5e44428e3a92:/home/web/media/packages/2020#
```

**Comment by timlinux** [comment]
I made a temporary fix by doing this:

```
root@5e44428e3a92:/home/web/media/packages/2020# chmod go+r *
root@5e44428e3a92:/home/web/media/packages/2020# ls -l
```

**Comment by timlinux** [comment]
@sumandari so for now your tasks are

- [ ] 
- [ ] Get the sftp backup running locally so that you also have a local backup
- [ ] Monitor the incoming backups to see if the problem arises again
- [ ] If it does please investigate why some plugins are not written with og+r permissions

**Comment by timlinux** [comment]
```
mirror: Access failed: Permission denied (/mnt/HC_Volume_4113275/lost+found)
Total: 58 directories, 149 files, 0 symlinks
To be removed: 5 directories, 53 files, 0 symlinks
2 errors detected
Source : /mnt/HC_Volume_4113256/packages
Destination : static
Objective : DOWNLOAD
mirror: Access failed: Permission denied (QTRANUS_fLWp7xJ.zip)  
mirror: Access failed: Permission denied (mesma-1.0.8-qgis.zip)
mirror: Access failed: Permission denied (networks_raGT588.zip)
mirror: Access failed: Permission denied (pat_v1.0.2.zip)
mirror: Access failed: Permission denied (pat_v1.0.2_z0nIRHD.zip)
mirror: Access failed: Permission denied (timeseriesviewerplugin.1.15.20201124T165509.master.zip)
Total: 7 directories, 20813 files, 0 symlinks
New: 6 files, 0 symlinks
To be removed: 13 directories, 53 files, 0 symlinks
7 errors detected
```

Problem still exists @sumandari

**Comment by sumandari** [comment]
this is what I got when run backup script
```
mirror: Access failed: Permission denied (/mnt/HC_Volume_4113275/lost+found)                    
Total: 58 directories, 150 files, 0 symlinks
New: 1 file, 0 symlinks
24191032 bytes transferred in 3708 seconds (6.4 KiB/s)
2 errors detected
Source : /mnt/HC_Volume_4113256/packages
Destination : static
Objective : DOWNLOAD
mirror: Access failed: Permission denied (QTRANUS_fLWp7xJ.zip)       
mirror: Access failed: Permission denied (UMEP_T3zREYG.zip)
mirror: Access failed: Permission denied (mesma-1.0.8-qgis.zip)
mirror: Access failed: Permission denied (networks_raGT588.zip)
mirror: Access failed: Permission denied (pat_v1.0.2.zip)
mirror: Access failed: Permission denied (pat_v1.0.2_z0nIRHD.zip)
mirror: Access failed: Permission denied (timeseriesviewerplugin.1.15.20201124T165509.master.zip)
Total: 7 directories, 20830 files, 0 symlinks
New: 7 files, 0 symlinks
To be removed: 0 directories, 2 files, 0 symlinks
8 errors detected
```

**Comment by timlinux** [comment]
```
Source : /mnt/HC_Volume_4113256/packages
Destination : static
Objective : DOWNLOAD
mirror: Access failed: Permission denied (QTRANUS_fLWp7xJ.zip)  
mirror: Access failed: Permission denied (UMEP_T3zREYG.zip)
mirror: Access failed: Permission denied (enmapboxplugin.3.7.20201202T164502.master.zip)
mirror: Access failed: Permission denied (mesma-1.0.8-qgis.zip)
mirror: Access failed: Permission denied (networks_raGT588.zip)
mirror: Access failed: Permission denied (pat_v1.0.2.zip)
mirror: Access failed: Permission denied (pat_v1.0.2_z0nIRHD.zip)
mirror: Access failed: Permission denied (spectral-libraries-1.1.0-qgis.zip)
mirror: Access failed: Permission denied (timeseriesviewerplugin.1.15.20201124T165509.master.zip)
Total: 7 directories, 20881 files, 0 symlinks
New: 9 files, 0 symlinks
```

**Comment by sumandari** [comment]
the fix #132 doesn't work. We need another solution.

**Comment by Xpirix** [comment]
The permission issue doesn't show anymore when running the backup script. About the duplicated files in static and media folders, the PR #340 should fix it.
